ArduCopter/motors.cpp

```diff
@@ -176,10 +180,26 @@ void Copter::motors_output()
         motors->output();
     }
 
+    // implement motor kill for parachute testing
+    for (uint8_t i = 0; i < MOTOR_COUNT; ++i) {
+        if (motor_kill_all || motor_kill[i]) {
+            hal.rcout->write(i, 1000);
+        }
+    }
+
     // push all channels
     SRV_Channels::push();
 }
 
+void set_motor_kill(uint8_t motor_num, bool kill)
+{
+    if (motor_num == 0xFF) {
```

Comment:
Is there a #define for 0xFF?

Reply:
My guess is no...this change was cherry-picked from Tridge's changes so I didn't write this myself. I suggest we define some of these in the future